OWML.Patcher/GameVersionReader.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Text;
+using System.Text.RegularExpressions;
+
+namespace OWML.Patcher
+{
+    public class GameVersionReader
+    {
+        private readonly BinaryPatcher _binaryPatcher;
+
+        private const int PlayerSettingsSector = 0;
+        private const string VersionPattern = @"\d+(\.\d+){3,}";//can handle 3-infinite dots
+
+        public GameVersionReader(BinaryPatcher binaryPatcher)
+        {
+            _binaryPatcher = binaryPatcher;
+        }
+
+        public string GetGameVersion()
```

Comment:
since you're reading *all* bytes, whats the real point of using the binart patcher? this looks so similar to #280

Reply:
I read only one sector